src/main/java/org/llorllale/cactoos/matchers/TextMatcher.java

```diff
@@ -37,6 +37,15 @@
  *
  * @since 1.0.0
  * @checkstyle ProtectedMethodInFinalClassCheck (200 lines)
+ * @todo #165:30m Add a constructor to TextMatcher taking a Func<String, Boolean> in order to simplify this TextMatcher:
+ *  new TextMatcher(
+ *  ____new MatcherOf<>(
+ *  ________(String act) -> act.endsWith(text.asString()),
+ *  ____________desc -> desc.appendText(text.asString()),
+ *  ____________(actual, desc) -> desc.appendValue(actual)
+ *  ____),
+ *  ____"Text ending with "
+ *  )
```

Comment:
@ruca905 a few things here:
- is it really needed to add those `_`?
- can you replace the example with the desired implementation instead of the current implementation
- can you also cite the name of the class this is about
- can you add something to say this should be done for all classes based on TextMatcher